gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "0.45.29",
+  "version": "0.45.28",
```

Comment:
This should be a minor version as it's got new functionality

Reply:
I'll update it before it gets merged.

Reply:
> I'll update it before it gets merged.

There are some differences to how the example pages should be structured in the newer version.  Please merge it now so I can then comment on what changes need to be made there.

Reply:
Master is merged in now. It looks the same as it did before to me?

Reply:
Right, now that you have the newest I can suggest that you use the new and improved `GalleryExampleTile` that ensure the desired structure for all gallery examples, and the appropriate tile subsection DOM and classes as demonstrated on the nx-tile page.